test/e2e/suite/plugin/verify.go

```diff
@@ -0,0 +1,37 @@
+package plugin
+
+import (
+	"fmt"
+
+	. "github.com/notaryproject/notation/test/e2e/internal/notation"
+	"github.com/notaryproject/notation/test/e2e/internal/utils"
+	. "github.com/notaryproject/notation/test/e2e/suite/common"
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("notation plugin verify", func() {
+	It("with basic case", func() {
+		Skip("notation plugin version verification is not correct.")
+		Host(BaseOptions(), func(notation *utils.ExecOpts, artifact *Artifact, vhost *utils.VirtualHost) {
+			// setup plugin and plugin-key
+			vhost.SetOption(AddPlugin(NotationE2EPluginPath))
+			notation.Exec("key", "add", "plugin-key", "--id", "key1", "--plugin", "e2e-plugin",
+				// add pluginConfig to enable generating envelope capability and update extended attribute
+				"-c", fmt.Sprintf("%s=true", CapabilityEnvelopeGenerator),
```

Comment:
We should add tests here on verification plugins with capabilities CapabilityTrustedIdentityVerifier, CapabilityRevocationCheckVerifier

Reply:
CapabilityTrustedIdentityVerifier and CapabilityRevocationCheckVerifier are enabled by default. So we don't need to specify to use them.

Reply:
Yes but we need to add tests associated with those capabilities.

Reply:
Yes, we have test case in suite/plugin/verify.go for those capabilities.